src/pages/_app.jsx

```diff
@@ -135,6 +170,7 @@ WrappedApp.getInitialProps = async ({ Component, ctx }) => {
     throw e;
   }
 };
+/* eslint-enable global-require */
```

Comment:
why is that?

Reply:
This is because we have to dynamically load SSR-only packages from the function directly using `require()` instead of relying on `import` statements in the top of `_app.jsx` -- those ones get automatically transpiled and inserted into the bundles that the client receives on the browser, so they can't contain stuff that can only be run on the server.

Linters generally don't like it when you `require()` from anywhere but the top of the file, which is sensible, but in this case it is necessary.

Reply:
makes sense Marcell, thanks for explaining!